vies/forms/fields.py

```diff
@@ -18,7 +18,7 @@ class VATINField(forms.MultiValueField):
     def __init__(self, choices=VIES_COUNTRY_CHOICES, *args, **kwargs):
         max_length = kwargs.pop('max_length', VATIN_MAX_LENGTH)
 
-        kwargs['widget'] = self.widget(choices=choices)
+        kwargs['widget'] = kwargs.get('widget', self.widget)(choices=choices)
```

Comment:
I would recommend to simply use setdefault here
```suggestion
        kwargs.setdefault('widget', self.widget(choices=choices))
```

Reply:
with setdefault, it would not construct the widget with choices if only class is passed...